src/main/java/com/pnudev/librarysystem/exception/handler/GlobalHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.pnudev.librarysystem.exception.handler;
+
+import com.pnudev.librarysystem.dto.ErrorDTO;
+import com.pnudev.librarysystem.exception.DeleteFailedException;
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.List;
+
+@RestControllerAdvice
+public class GlobalHandler {
```

Comment:
`GlobalExceptionHandler` would be more suitable

Reply:
Renamed